stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/channel/list/viewmodel/ChannelListViewModel.kt

```diff
@@ -31,8 +33,9 @@ public class ChannelListViewModel(
     private val chatDomain: ChatDomain = ChatDomain.instance(),
     private val filter: FilterObject = Filters.and(
         eq("type", "messaging"),
-        Filters.`in`("members", listOf(chatDomain.currentUser.id)),
-        Filters.or(Filters.notExists("draft"), Filters.ne("draft", true)),
+        `in`("members", listOf(chatDomain.currentUser.id)),
+        or(Filters.notExists("draft"), ne("draft", true)),
+        eq("hidden", false)
```

Comment:
This filter won't work if extra data is null (backend issue, already reported). To workaround that, you can combine it with 'notExists' filter (just like for draft).
One more thing - shouldn't we also add this to VieModelFactories?